Running make infrastructure fails with error

$ make infrastructure
# Get the modules, create the infrastructure.
terraform init && terraform get && terraform apply -auto-approve
Initializing modules...
- module.openshift

Initializing provider plugins...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.aws: version = "~> 2.3"
* provider.local: version = "~> 1.2"
* provider.template: version = "~> 2.1"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
- module.openshift

Error: module.openshift.aws_route53_zone.internal: "vpc_id": [REMOVED] use 'vpc' configuration block instead


make: *** [infrastructure] Error 1

--- a/modules/openshift/07-dns.tf
+++ b/modules/openshift/07-dns.tf
@@ -5,7 +5,9 @@
 resource "aws_route53_zone" "internal" {
   name = "openshift.local"
   comment = "OpenShift Cluster Internal DNS"
-  vpc_id = "${aws_vpc.openshift.id}"
+  vpc {
+    vpc_id = "${aws_vpc.openshift.id}"
+  }
   tags {
     Name    = "OpenShift Internal DNS"
     Project = "openshift"

